### DISCUSSION TITLE
New requirement.

### DISCUSSION THREAD

**Discussion by gaochuang** [open-telemetry/opentelemetry-specification#3568] [Ideas -> please open issues instead for feature requests!]
Hi,
 We have a new requirements. New requirement could fix this issue. For details, please refer to the discussion in this link

https://github.com/open-telemetry/opentelemetry-cpp/issues/2199

**Comment by gaochuang** [comment]
@lalitb Am I adding a new requirement here? Thanks.

**Reply by trask** [reply]
hi @gaochuang, if you want to propose a new specification requirement, please open an issue in this repository
